test/init/language/css/init-language-css.test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
+const { runPromptWithAnswers } = require('../../../utils/test-utils');
+const firstPrompt = 'Will your application have multiple bundles?';
+
+const ENTER = '\x0D';
+const DOWN = '\x1B\x5B\x42';
+const genPath = path.join(__dirname, 'test-assets');
+
+jest.setTimeout(400000);
```

Comment:
Why number is so big?

Reply:
My bad (my system is slow so I increased it, but I forget to update it before pushing commit in PR branch 😅), will update it in next commit.